apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -489,6 +489,24 @@ export const storeDocumentVersion = async (request, response) => {
 		throw new BackOfficeAppError(`Document not found: guid ${guid}`, 404);
 	}
 
+	if (documentVersionMetadataBody.transcript) {
+		const transcriptReference = documentVersionMetadataBody.transcript;
+
+		const transcriptDocument = await documentRepository.getByReferenceRelatedToCaseId(
+			transcriptReference,
+			+caseId
+		);
+
+		if (!transcriptDocument) {
+			const transcriptErrorMsg = `Transcript document not found: reference ${transcriptReference}`;
+			response.status(404).send({ errors: { transcript: transcriptErrorMsg } });
+
+			throw new BackOfficeAppError(transcriptErrorMsg, 404);
```

Comment:
```suggestion
			throw new BackOfficeAppError(transcriptErrorMsg, 404);
```

If you throw a `BackOfficeAppError` it should implicitly respond with a 404 and the error message you give it, so you don't need to do it explicitly.